quickstack/manifests/compute_common.pp

```diff
@@ -284,4 +287,36 @@
            nova_uuid     => $nova_uuid,
            rbd_key       => $rbd_key,
         }
+#Custimization for ruby. Ruby is necessary dependency of sensu
```

Comment:
I think we already have ruby gems. Can you please tell why do we need updated ruby gems.